test/Driver/sanitizers.swift

```diff
@@ -1,20 +1,15 @@
+// REQUIRES: CPU=x86_64
```

Comment:
No need for this here since nothing's actually getting run and everything has an explicit target.